catkin_home/src/action_selectors/scripts/DeepSpeech.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+'''
+This script creates the node `DeepSpeech` that taking voice audio from topic
+`UsefulAudioDeepSpeech`, does speech-to-text and publishes the resulting text.
+
+'''
+
+import rospy
+import scipy
+
+from audio_common_msgs.msg import AudioData
+from action_selectors.msg import RawInput
+from SpeechApiUtils import SpeechApiUtils
+from DeepSpeech.deploy.ros_server import ASRServer
+
+# Num of process for searching in the LM scorer.
+NUM_PROCESSES_BEAM_SEARCH = 2
+
+publisher = None
+asr_server = None
+
+
+def on_audio_callback(data):
+    rospy.loginfo("DeepSpeech computing...")
+    unicode_text = asr_server.bytes_speech_to_text(data.data)
+    text = unicode_text.encode('ascii', 'ignore')
+
+    if len(text) == 0 or text.isspace():
+       rospy.loginfo("Audio is empty")
+       return
+
+    rospy.loginfo("Voice audio said: \"{0}\".".format(text))
+
+    msg = RawInput()
+    
+    msg.isWoman = False
```

Comment:
You can put a TODO comment here, where we need to identify if a man or woman said the command.